app/src/main/java/com/example/habitsmasher/DaysTracker.java

```diff
@@ -147,6 +120,49 @@ public String getDays(){
     }
 
 
+    /**
+     * Takes a string with the days and converts them to days of the week.
+     * The string takes the format "day1 day2 ... day7", where each day is the first 2 letters of that day, separated by a space. The string can be in any order.
+     * EX) "MO FR WE TH"
+     *
+     * @param days The days to select
+     */
+    public void setDays(String days){
```

Comment:
New method, takes the original functionality of the string based constructor and extracts it to its own method. Could potentially prove useful down the line as well.